imports/components/@enhancers/security-roles/__tests__/index.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable */
+
+import { Component } from "react";
+import { shallow } from "enzyme";
+import { shallowToJson } from "enzyme-to-json";
+import { SECURITY_ROLES_QUERY, canSee, authorized } from "../";
+
+it("has a valid query", () => {
+  expect(SECURITY_ROLES_QUERY).toMatchSnapshot();
+});
+
+it("is renders a wrapper function", () => {
+  expect(canSee(["Staff"])).toMatchSnapshot();
```

Comment:
Yea, I talked everything over with @jbaxleyiii and I have a better idea of how to test this so it'll be going away soon.